tests/contrib/openai_agents/test_openai.py

```diff
@@ -1778,3 +1778,22 @@ async def test_workflow_method_tools(client: Client):
                 execution_timeout=timedelta(seconds=10),
             )
             await workflow_handle.result()
+
+
+async def test_response_serialization(client: Client):
```

Comment:
nit: client unused (I'm planning to make CI fail on this category of type checker warning eventually)